ch01python/016functions.ipynb.py

```diff
@@ -219,37 +217,23 @@ def doubler(*sequence):
     return [x * 2 for x in sequence]
 
 
-# %%
-doubler(1, 2, 3)
-
-# %%
-doubler(5, 2, "Wow!")
+print(doubler(1, 2, 3, "four"))
 
 
 # %% [markdown]
-# ### Keyword Arguments
+# ## Keyword Arguments
 
 # %% [markdown]
-# If two asterisks are used, named arguments are supplied inside the function as a dictionary:
+#
+# If two asterisks are used, named arguments are supplied as a dictionary:
+#
+#
+#
 
 # %%
 def arrowify(**args):
     for key, value in args.items():
-        print(f"{key} -> {value}")
-
-arrowify(neutron="n", proton="p", electron="e")
+        print(key + " -> " + value)
 
 
-# %% [markdown]
-# These different approaches can be mixed:
-
-# %%
-def somefunc(a, b, *args, **kwargs):
-    print("A:", a)
-    print("B:", b)
-    print("args:", args)
-    print("keyword args", kwargs)
-
-
-# %%
-somefunc(1, 2, 3, 4, 5, fish="Haddock")
```

Comment:
This is probably worth having, as showing the mix of arguments and keyword arguments.